src/components/Treemap.tsx

```diff
@@ -334,6 +335,50 @@ export default class Treemap extends React.Component<
     );
   }
 
+  attrWidth = ({ data, x1, x0 }) => {
+    const {
+      xScale,
+    } = this.state;
+    if (data.attribution) {
+      const {value} = data.attribution;
+      const cellWidth = xScale(x1) - xScale(x0);
+      return (cellWidth * value);
+    }
+    return 0;
+  }
+
+  renderAttribution = d => {
+    const {
+      onMouseMove,
+      onMouseLeave,
+      onMouseOver,
+      strokeWidth,
+    } = this.props;
+
+    const {
+      yScale,
+    } = this.state;
+
+    const color = (d.data.attribution && d.data.attribution.color) ? d.data.attribution.color : null;
```

Comment:
A couple notes here:
Unless there is a problem with `undefinded` vs `null` (which I don't think there is in this case) I believe a cleaner way to write this would be:

`const color = d.data.attribution && d.data.attribution.color;`

Since it's being used specifically for `fill` would it be best to call it `fill`?
Lastly, we don't have to do it now, but it may be good to offer a way to modify the color besides having a nested `color` value. We'll see it more clearly looking at other IHME-UI components but IHME-UI allows a user to color by a `colorScale` prop, a `fill`/`color` prop, or by the `style` prop. Again, let's tackle that later, but it's something to think about since this implementation demands the data be in a very particular shape in order to ensure it gets colored. Thoughts?